fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/domain/SinglePrefer.java

```diff
@@ -41,7 +41,13 @@
      * @throws ParseException if parse exception occurred
      */
     public SinglePrefer(final String header) throws ParseException {
-        preferTags.addAll(HttpHeaderReader.readList(PreferTag::new, header));
+        preferTags.addAll(stream(header.split(",")).map(h -> {
+            try {
+                return new PreferTag(h);
+            } catch (final ParseException e) {
+                throw new IllegalArgumentException("Could not parse 'Prefer' header", e);
+            }
+        }).collect(toSet()));
```

Comment:
Hm, good eye.  Outside of testing, it looks like the only place SinglePrefers are instantiated is inside the jax-rs impl (by virtue of `MultiPrefer(final @HeaderParam("Prefer") Set<SinglePrefer> prefers)`.  It certainly doesn't care about exception types, so removing the `throws ParseException` and throwing a runtime exception should be perfectly fine in practice.  However, it also implies that the `throws ParseException` in MultiPrefer will now never occur.  Should I remove it there too?